websites/S/Stremio/presence.ts

```diff
@@ -0,0 +1,179 @@
+const presence = new Presence({
+		clientId: "969208766807547917"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video: HTMLMediaElement,
+	timestamp: [number, number],
+	pauseCheck: boolean,
+	search: HTMLInputElement,
+	genreSort: string,
+	title: string;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		},
+		page = window.location.href,
```

Comment:
```suggestion
		{ pathname } = document.location,
```

Reply:
maybe tell ur colleague to stop requesting if u want differently

Reply:
I don't mind how you do it, just destructure the object instead of making another var name for consistency